tests/pytests/test_empty.py

```diff
@@ -590,12 +590,6 @@ def testEmptyTextHash(env, idx, dialect):
         cmd_assert(env, cmd, expected)
         conn.execute_command('DEL', 'h2')
 
-        # ----------------------------- Fuzzy search ---------------------------
-        # We don't expect to get empty results in a fuzzy search.
```

Comment:
Shouldn't this still be valid for lower dialects?
Maybe it is worth to preserve the test under certain conditions?

Reply:
Only for dialect 1, this is checked later on separately